Implement titlecase iterator (#139)
artichoke/roe: generated/ucd/README.md

Reviewer: I think this file should be in `generated/README.md` instead of this sub folder.

Author: Fixed in 01d0ac0 too.

> Do not edit `generated/ucd/*.txt`. They were automatically generated by:

Reviewer: ```suggestion
Do not edit `generated/ucd/*.txt`. The files are automatically generated by:
```

Author: Thanks. Fixed in 01d0ac0.
